src/main/java/io/github/minecraftcursedlegacy/mixin/MixinEntityRegistry.java

```diff
@@ -0,0 +1,57 @@
+package io.github.minecraftcursedlegacy.mixin;
+
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityRegistry;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Accessor;
+import org.spongepowered.asm.mixin.gen.Invoker;
+
+import java.util.Map;
+
+@Mixin(EntityRegistry.class)
+public interface MixinEntityRegistry {
```

Comment:
we put accessors in a different mixin json with a different naming scheme

Reply:
and in a different package I think

Reply:
Done.